homeassistant/components/hvv_departures/sensor.py

```diff
@@ -58,15 +58,15 @@ class HVVDepartureSensor(SensorEntity):
     _attr_attribution = ATTRIBUTION
     _attr_device_class = SensorDeviceClass.TIMESTAMP
     _attr_icon = ICON
+    _attr_translation_key = "departures"
+    _attr_has_entity_name = True
+    _attr_extra_state_attributes = {}
```

Comment:
We shouldn't initialize a class attribute to a mutable object. That's error prone and rarely needed.

If this platform would have added more than one sensor entity, every time one entity updated its state attributes, all sensor instances would get that update to their state attributes dict.